spec/unit/util/windows/sid_spec.rb

```diff
@@ -15,7 +15,7 @@
   context "#octet_string_to_sid_object" do
     it "should properly convert an array of bytes for the local Administrator SID" do
       host = '.'
-      username = 'Administrator'
+      username = ENV['USERNAME']
```

Comment:
You should be able to use our helper method here, which calls the `[GetUserName](http://msdn.microsoft.com/en-us/library/windows/desktop/ms724432(v=vs.85).aspx)` Win32 API call under the hood.  Should be a bit more reliable than relying on ENV vars, which I don't know if AppVeyor monkeys with or not....

```ruby
Puppet::Util::Windows::ADSI.current_user_name
```